launch/sensor_trigger.launch.xml

```diff
@@ -4,11 +4,13 @@
   <arg name="gpio" default="51"/>
   <arg name="phase" default="0.0"/>
   <arg name="frame_rate" default="10.0"/>
+  <arg name="assigned_cpu" default="1"/>
```

Comment:
Different argument names between the launch file and the node (i.e., `assigned_cpu` vs. `cpu`) are the intended ones?
Because the launch argument differs from one mentioned in README, I am concerned that it might cause users confusion.

Reply:
Good point, I will edit! Thanks

Reply:
Fixed with 929c7d0cb5339ff7a705770a74f07ff4b42c898c

Reply:
Looks good to me. Thank you for your handling!